tools/tsp-client/src/commands.ts

```diff
@@ -247,7 +247,7 @@ export async function syncCommand(argv: any) {
 
   try {
     let emitterLockPath = getEmitterLockPath(getEmitterPackageJsonPath(repoRoot, tspLocation));
-    
+
```

Comment:
is there nothing that enforce formatting?

Reply:
Nope, I have to add it eventually. Tracking issue for that: https://github.com/Azure/azure-sdk-tools/issues/11073